### DISCUSSION TITLE
Your Orchard Core site hosted on DotNest (almost) never goes down

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#14268] [Show and tell]
[DotNest](https://dotnest.com), our public Orchard Core and Orchard 1 SaaS is the easiest way to host Orchard-based websites. And it's not just easy but reliable as well, with 99.99% availability in the last 3 months. Check out the details in our latest blog post here: https://dotnest.com/blog/your-dotnest-site-almost-never-goes-down

As an admin user, you can break your own site though. So if you did that and can't recover, just let me know and I can help you :D.

**Comment by rwxzig** [comment]
This is super cool, the fact that you are offering a free tier is awesome.

**Reply by Piedone** [reply]
Thank you!
